internal/server/resource/resource.go

```diff
@@ -25,6 +25,18 @@ import (
 // PVs to a number greater than max number of PVs for a human curated stat var.
 const nonHumanCuratedNumPv = 30
 
+// prefixes for pv definitions for required properties
+var requiredPVDefPrefixes = map[string]struct{}{
```

Comment:
nit: These are all technically not "required" (eg mq, md).  Might rename as "structured" or "known", or something

Reply:
done